### PR TITLE
log message should start with nginx-sxg-module:

### PR DESCRIPTION
**Pull request by kumagi**
Log should be verbose to detect the problem.

### CODE REVIEW

File: ngx_http_sxg_filter_module.c [hunk 1]
```diff
@@ -447,7 +445,8 @@ static bool copy_buffer_to_sxg_buffer(const ngx_http_request_t* req,
         const size_t buffer_tail = buf->size;
         if (buffer_tail + copy_size > limit) {
           ngx_log_error(NGX_LOG_ERR, req->connection->log, 0,
-                        "Too large buffer size required: %d bytes",
+                        "nginx-sxg-module: too large buffer size required "
```
Comment (gaul) [inline ngx_http_sxg_filter_module.c:448]: If you call `ngx_log_init("nginx-sxg-module")` during initialization does it centralize this prefix?

Comment (kumagi) [reply ngx_http_sxg_filter_module.c:448]: Unfortunately, No.
`ngx_log_init(prefix)` modifies [global variable](
https://github.com/nginx/nginx/blob/master/src/core/ngx_log.c#L317-L326) and seems to be a function for application wide configuration with [command line flag](https://github.com/nginx/nginx/blob/master/src/core/nginx.c#L790).

Comment (kumagi) [reply ngx_http_sxg_filter_module.c:448]: `ngx_log_error` is a macro which wraps `ngx_log_error_core`.
I'm not sure to re-wrap this with another (prefix appending) macro helps readability.

**Comment by kumagi** [comment]
Thank you for review!

**Pull request merged**